app/controllers/insured/members_selection_controller.rb

```diff
@@ -14,6 +14,8 @@ def new
       end
     end
 
+    # params = {"change_plan"=>"change_plan", "coverage_kind"=>"dental", "employee_role_id"=>"5b46d6b9aea91a4acd7bf718", "event"=>"make_changes_for_dental", "hbx_enrollment_id"=>"6406366b73722352a12b277d", "person_id"=>"5b46d6b9aea91a4acd7bf714"}
```

Comment:
if not needed, can you please remove this dead code ? , or if its an example, can you add a header explaining how it would be useful